Microsoft.HealthVault/Clients/Deserializers/ThingDeserializer.cs

```diff
@@ -19,11 +21,24 @@ namespace Microsoft.HealthVault.Clients.Deserializers
 {
     internal class ThingDeserializer : IThingDeserializer
     {
+        private readonly Guid _applicationSpecificId = new Guid("a5033c9d-08cf-4204-9bd3-cb412ce39fc0");
```

Comment:
this is application specific type id ..

---
In reply to: [123100329](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/62#discussion_r123100329) [](ancestors = 123100329)